src/components/table/column-picker/column-picker.component.ts

```diff
@@ -186,7 +186,9 @@ export class ColumnPickerComponent implements OnChanges {
 
     /** Update when reordering has occurred */
     onReorder(): void {
-        this.selectedChange.emit(this.selected);
+        setTimeout(() => {
```

Comment:
I think using setTimeout here is a bit of a hack. Looking at the source, I guess that the problem is that `reorderEnd` emits before `reorderableModelChange`. Perhaps you can emit `selectedChange` directly after `selected` has been updated instead.